Tests/Fixtures/MetsDocument/mets_with_pages.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mets:mets xmlns:mets="http://www.loc.gov/METS/"
+           xmlns:xlink="http://www.w3.org/1999/xlink"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.loc.gov/standards/mets/mets.xsd">
+    <mets:fileSec>
+        <mets:fileGrp USE="FULLTEXT">
+            <mets:file ID="FULLTEXT_0001" MIMETYPE="application/pdf">
+                <mets:FLocat LOCTYPE="URL" xlink:href="fulltext_0001.xml"/>
```

Comment:
Is it intentional that this (and `FULLTEXT_0002` and `FULLTEXT_0004`) is not a valid URL? Looks like only the entry for `FULLTEXT_0003` is used in the tests which also is the only one with a valid URL.

If we do not need invalid METS for testing, we should fix this and use valid URLs for every FULLTEXT entry.